javacord-api/src/main/java/org/javacord/api/entity/server/Server.java

```diff
@@ -3515,7 +3515,7 @@ default CompletableFuture<Sticker> requestStickerById(String id) {
         try {
             return requestStickerById(Long.parseLong(id));
         } catch (NumberFormatException exception) {
-            throw new IllegalArgumentException("The ID must be a number.");
+            return requestStickerById(-1);
```

Comment:
```suggestion
            CompletableFuture<Sticker> future = new CompletableFuture<>();
            future.completeExceptionally(exception);
            return future;
```
Same as above. After the switch to Java 11 we can also do `return CompletableFuture.failedFuture(exception);`